imageio/plugins/simpleitk.py

```diff
@@ -13,23 +13,50 @@
 
 
 def load_lib():
-    global _itk
+    global _itk, _read_function, _write_function
     try:
-        import SimpleITK as _itk
+        import itk as _itk
+
+        _read_function = _itk.imread
+        _write_function = _itk.imwrite
     except ImportError:
-        raise ImportError(
-            "SimpleITK could not be found. "
-            "Please try "
-            "  easy_install SimpleITK "
-            "or refer to "
-            "  http://simpleitk.org/ "
-            "for further instructions."
-        )
+        try:
+            import SimpleITK as _itk
+
+            _read_function = _itk.ReadImage
+            _write_function = _itk.WriteImage
+        except ImportError:
+            raise ImportError(
+                "itk could not be found. "
+                "Please try "
+                "  python -m pip install itk "
+                "or refer to "
+                "  https://itkpythonpackage.readthedocs.io/ "
```

Comment:
I guess we want to refer to both `itk` and `simpleitk` here? 

BTW: hooray for binary wheels on pypi!

Reply:
@almarklein Good idea -- I will also add the updated simpleitk installation instructions.